### PR TITLE
Finalised version of Ben's code

### PR DESCRIPTION
**Pull request by edbennett**
Ben's latest changes as PRed to my repo.

### CODE REVIEW

File: migrate/admin.py [hunk 1]
```diff
@@ -1,3 +0,0 @@
-from django.contrib import admin
-
```
Comment (markgdawson) [inline migrate/admin.py:2]: @bjthorpe @edbennett do either of you know what these deletions are? Just checking it isn't merge weirdness....

Comment (edbennett) [reply migrate/admin.py:2]: Ben accidentally left a test application he was using to learn Django in the original version of this PR which you pulled, and then removed it for the final PR.

Comment (markgdawson) [reply migrate/admin.py:2]: Looks like merge weirdness from a previous partial merge from your branch.

I'll assume this is OK.

**Review by markgdawson — APPROVED** [review]
Pow!

**Pull request merged**